### PR TITLE
feature: Match new app navigation

### PR DESCRIPTION
**Pull request by mrfyda**
Note the path changes

### CODE REVIEW

**Review by rtfpessoa — APPROVED** [review]

File: docs/metrics/lead-time-reviews.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Lead time sub-metrics & Reviews metrics
```
Comment (pauloribeiro-codacy) [inline docs/metrics/lead-time-reviews.md:1]: Could this title be simplified to:

```suggestion
# Lead time and review metrics
```

File: docs/metrics/lead-time-reviews.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+This metric is calculated as the time between the first commit in the pull request's branch and the pull request being created.
+
+### Time to review
+
+The time spent in the code review process. Once a pull request is opened, how long does it take to merge?
+
+This metric is calculated as the time between the pull request being created and the pull request being merged.
+
+## Reviews metrics
```
Comment (pauloribeiro-codacy) [inline docs/metrics/lead-time-reviews.md:27]: As in "pull request **review** metrics":

```suggestion
## Review metrics
```

File: docs/metrics/accelerate.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Calculated metrics
+# Accelerate metrics
```
Comment (pauloribeiro-codacy) [inline docs/metrics/accelerate.md:1]: Like this, the page will have two top-level headings, which is [against the rules](https://www.npmjs.com/package/remark-lint-no-multiple-toplevel-headings). :smile: Instead, we should keep a top-level heading to act as the page title, and then we can have two sections for each group of metrics.

Comment (pauloribeiro-codacy) [reply docs/metrics/accelerate.md:1]: Anyway, I really like that we're promoting the "sub-metrics" to its own section on par with the Accelerate metrics, it makes sense to:

* Give them more visibility
* Prepare the structure of the page for subsequent metrics that we might add without incurring in too much nesting

**Review by pauloribeiro-codacy — CHANGES_REQUESTED** [review]

File: docs/metrics/accelerate.md (hunk 3 above)
Comment (mrfyda) [reply docs/metrics/accelerate.md:1]: > Like this, the page will have two top-level headings, which is [against the rules](https://www.npmjs.com/package/remark-lint-no-multiple-toplevel-headings). 😄 Instead, we should keep a top-level heading to act as the page title, and then we can have two sections for each group of metrics.

Even with the plugin, I wasn't able to understand what you meant. Sorry. 😅 
Let's talk sync.

Comment (pauloribeiro-codacy) [reply docs/metrics/accelerate.md:1]: It's fixed now. :raised_hands: Perhaps it was already fixed when I wrote my comment, and I was seeing an outdated version of the file, hence the confusion?

**Review by pauloribeiro-codacy — APPROVED** [review]

**Pull request merged**